app/src/modules/unified/contents/VerificationStep1/Instructions.tsx

```diff
@@ -0,0 +1,82 @@
+import { Button, ButtonType, testIdWithKey, useTheme } from '@hyperledger/aries-bifold-core'
+import { useTranslation } from 'react-i18next'
+import { Text, StyleSheet, ScrollView, Image, View, useWindowDimensions } from 'react-native'
+import { SafeAreaView } from 'react-native-safe-area-context'
+
+const pagePadding = 24
+
+type ContentProps = {
+  goToScan: () => void
+  goToManualSerial: () => void
+}
+
+const InstructionsContent: React.FC<ContentProps> = ({ goToScan, goToManualSerial }: ContentProps) => {
+  const { t } = useTranslation()
+  const { ColorPallet, TextTheme } = useTheme()
+  const { width } = useWindowDimensions()
+
+  const styles = StyleSheet.create({
+    pageContainer: {
+      height: '100%',
+      justifyContent: 'space-between',
+      backgroundColor: ColorPallet.brand.secondaryBackground,
+    },
+    scrollView: {
+      flex: 1,
+      padding: pagePadding,
+    },
+    controlsContainer: {
+      marginBottom: 20,
+      marginTop: 'auto',
+      marginHorizontal: 20,
+      position: 'relative',
+    },
+    image: {
+      width: width - pagePadding * 2,
+      height: (width - pagePadding * 2) * 0.67,
+      marginBottom: 24,
+    },
+    heading: {
+      ...TextTheme.headingThree,
+      marginBottom: 16,
+    },
+    paragraph: {
+      ...TextTheme.normal,
+      marginBottom: 8,
+    },
+  })
+
+  return (
+    <SafeAreaView style={styles.pageContainer} edges={['bottom', 'left', 'right']}>
+      <ScrollView contentContainerStyle={styles.scrollView}>
+        <Image
+          source={require('../../assets/img/highlight_serial_barcode.png')}
```

Comment:
Would this work? Using require in-line seems a bit odd to me.

```
import highlightSerialBarcode from '../../assets/img/highlight_serial_barcode.png';

// ... in your component ...
<Image source={highlightSerialBarcode} /> 
```

Reply:
Inline require seems like the way to do according to the docs: https://reactnative.dev/docs/images

I get errors if I move the require to the top of the file or if I turn it into an import statement